README.md

```diff
@@ -144,6 +150,11 @@ We also support lists of prompt/completion pairs within a `.jsonl` file. This gu
 
 If the JSON objects in your `.jsonl` contain keywords other than **prompt** and **completion**, refer to the `prompt_keyword` and `completion_keyword` flags [below](#prompt_keyword)
 
+### Preparing data for Chat or Instruct Models
+Many chat and instruct models require very specific formatting to input multi turn conversations for training and inference using the [tokenizer.apply_chat_template function](https://huggingface.co/docs/transformers/main/en/chat_templating). We support tokenizing your data in this format, to do so please prepare your data in jsonl format as specified above, and then inlcude the `--apply_chat_template` flag to automatically prepare your data in this format.
```

Comment:
```suggestion
Many chat and instruct models require very specific formatting to input multi turn conversations for training and inference. The [tokenizer.apply_chat_template function](https://huggingface.co/docs/transformers/main/en/chat_templating) easily adapts your jsonl data to this format. To use this feature, prepare your data in jsonl format as specified above, and then include the `--apply_chat_template` flag to automatically prepare your data in this format.
```

Light wordsmithing - the first sentence makes it seem like using the `apply_chat_template` requires a specific formatting, when in reality - IIUC - it's the chat/instruction tuned models that require this format and this feature helps prepare your data for that.